index.html

```diff
@@ -2,9 +2,33 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <link href="styles.css" rel="stylesheet"/>
     <title>Tech It Easy Dashboard</title>
 </head>
 <body>
-
-</body>
-</html>
\ No newline at end of file
+<p>TECH IT EASY DASHBOARD</p>
+<div class="item">
+    <div class="item-a">
+        <h2>TV's to Sell</h2>
+        <h3 id="numbertoSell"> 0</h3>
+    </div>
+    <div class="item-b">
+        <h2>Revenue to expect</h2>
+        <h3 id="RE"> 0</h3>
+    </div>
+    <div class="item-c">
+        <h2>Revenue total</h2>
+        <h3 id="RT">0 </h3>
+        </div>
+    <div class="item-d">
+    <ul id = "product">
+        <li class = "itemTV">geen appendchild weergave</li>
```

Comment:
Is er een reden waarom `geen appendchild weergave` hier staat? Normaliter zou ik zeggen dat je geen list items hoeft toe te voegen, omdat je die codematig pas gaat toevoegen